packages/jellyfish-api-core/__tests__/category/loan/getCollateralToken.test.ts

```diff
@@ -28,15 +28,15 @@ describe('Loan getCollateralToken', () => {
     const collateralTokenId1 = await testing.container.call('setcollateraltoken', [{
       token: 'AAPL',
       factor: new BigNumber(0.5),
-      priceFeedId: priceFeedId1
+      priceFeedId: 'AAPL/USD'
```

Comment:
Consider expliciting the fact that priceFeedId is `${priceFeed.token}/${priceFeed.currency}`
```
       const priceFeed = {
         token: 'AAPL',
         currency: 'USD'
       }
       
       await testing.container.call('appointoracle', [await testing.generateAddress(), [priceFeed], 1])

       const collateralTokenId1 = await testing.container.call('setcollateraltoken', [{
          token: 'AAPL',
          factor: new BigNumber(0.5),
          priceFeedId: `${priceFeed.token}/${priceFeed.currency}`
       }])

Reply:
I used 'AAPL/USD' for better readability